input/resources/valueset-common-au-recorded-sex-or-gender-source-document-jurisdiction.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ValueSet xmlns="http://hl7.org/fhir">
+    <id value="common-au-recorded-sex-or-gender-source-document-jurisdiction" />
+    <meta>
+        <profile value="http://hl7.org/fhir/StructureDefinition/shareablevalueset" />
+    </meta>
+    <extension url="http://hl7.org/fhir/StructureDefinition/structuredefinition-fmm">
+        <valueInteger value="0"/>
+    </extension>
+    <url value="http://terminology.hl7.org.au/ValueSet/common-au-recorded-sex-or-gender-source-document-jurisdiction" />
+    <identifier>
+        <system value="urn:ietf:rfc:3986"/>
+        <value value="urn:oid:2.16.840.1.113883.2.3.4.2.4.33"/>
+    </identifier>
+    <name value="CommonAURecordedSexOrGenderSourceDocumentType" />
+    <title value="Common AU Recorded Sex or Gender (RSG) Source Document Jurisdiction" />
+    <status value="draft" />
+    <experimental value="false"/>
+    <description value="This value set includes codes that can be used to represent the jurisdiction of source documents associated with a Recorded Sex or Gender (RSG) value in an Australian context." />
```

Comment:
Change the prelim text we had in the draft files "This value set includes codes that can be used to represent the jurisdiction of source documents associated with a Recorded Sex or Gender (RSG) value in an Australian context."

To be something like "The Common AU Recorded Sex or Gender (RSG) Source Document Jurisdiction value set includes codes can be used to identify the jurisdiction that issued the source document where the sex or gender value is recorded."

Reply:
Done.